src/back/auth/src/api/token_introspect.cpp

```diff
@@ -12,14 +12,21 @@ TokenIntrospect::TokenIntrospect(
 	const components::ComponentContext& ctx)
 	: server::handlers::HttpHandlerBase{conf, ctx}
 	, _s{ctx.FindComponent<Service>()}
+	, _mapHttpMethodToSchema{LoadSchemas(kName, _s.GetJSONSchemasPath())}
 {}
 
 std::string TokenIntrospect::HandleRequestThrow(
 	const server::http::HttpRequest& req,
 	server::request::RequestContext&) const
 {
-	auto& token = req.GetCookie(Consts::SessionCookieName);
-	const std::string& userAgent = req.GetHeader(http::headers::kUserAgent);
+	const formats::json::Value body;
+	const auto params = ValidateRequest(_mapHttpMethodToSchema, req, body);
+
+	std::string token;
+	if (params.HasMember(Consts::SessionCookieName))
+		token = params[Consts::SessionCookieName].As<std::string>();
+
+	const std::string& userAgent = params[http::headers::kUserAgent].As<std::string>();
```

Comment:
Тут не указываем ссылку, потому что функция возвращает новый объект, а не объект по ссылке.
```suggestion
	const auto userAgent = params[http::headers::kUserAgent].As<std::string>();
```

Reply:
Готово.